pkg/kubelet/config/config.go

```diff
@@ -76,7 +76,7 @@ func (c *PodConfig) Channel(source string) chan<- interface{} {
 	return c.mux.Channel(source)
 }
 
-func (c *PodConfig) SourceSeen(source string) bool {
+func (c *PodConfig) IsSourceSeen(source string) bool {
```

Comment:
If we're fixing nits... please add a godoc comment, since this is a public function :)